gruntfile.js

```diff
@@ -89,6 +89,16 @@ module.exports = function(grunt) {
         tagMessage: 'Version %VERSION%',
         push: false
       }
+    },
+    karma: {
```

Comment:
Generally, I prefer to have all tasks together in one place for convenience. Since we are already using grunt, then it looks a good place for me to have them. I can also combine multiple tasks together and run them concurrently. At the end of the day, if you are thinking of putting grunt aside in the future, we can add them as npm scripts.